### ISSUE TITLE
Mark monteCarloMed as defunct?

### ISSUE THREAD

**Issue by LukasWallrich** [simsem/semTools#97]
I just tried to run old code and got the error `Error: 'monteCarloMed' is not an exported object from 'namespace:semTools'`, which made me search for typos etc. After a while, I found the NEWS that it has been replaced. Would it make sense to mark the old function as defunct rather than just remove it, so that it directs the user to the new function? AFAIK the following code should do that.

```
monteCarloMed= function(...) {
  .Defunct("monteCarloCI")

}
```

**Comment by LukasWallrich** [comment]
Two more comments on the new function (which is brilliant in extracting the information directly from lavaan objects - thanks!): 

- would it make sense to have an option to return both the summary and the samples? I understand that returning data.frames rather than lists is tidier, but in my use case, that requires me to run the function twice to achieve what monteCarloMed did in one go (I very much appreciate the new clean variable names, though).

- is the change from 20k to 200k nRep intended? Based on my understanding, 20k should be sufficient, and changing the default might introduce errors in reporting?

**Comment by TDJorgensen** [comment]
Woops!  I was in a rush to push a new version to CRAN before I taught an online SEM course in July, and I forgot to deprecate `monteCarloMed()`.  Thanks for bringing it to my attention.  I just used `.Defunct()` and added an alias so that `?monteCarloMed` also takes users to the `monteCarloCI()` help page.

> would it make sense to have an option to return both the summary and the samples?

Sure, you might want to add additional user-defined parameters or calculate more precise CIs using the highest-density region.  I changed the argument `return.samples=` to `append.samples=`, which will return a `list` of `data.frame`s.  I also added an example to the help page showing how to obtain HDIs.

> is the change from 20k to 200k nRep intended? 

oh, that was a typo, should have been `2e4` (now fixed).
